.github/workflows/release-publish.yml

```diff
@@ -229,7 +225,13 @@ jobs:
 
         * ${NUM_COMMITS} commits since ${LAST_TAG#nessie-}
         * Maven Central: https://search.maven.org/search?q=g:org.projectnessie+v:${RELEASE_VERSION}
-        * Docker Hub: https://hub.docker.com/r/projectnessie/nessie \`docker pull projectnessie/nessie:${RELEASE_VERSION}\`
+        * Docker Hub: https://hub.docker.com/r/projectnessie/nessie
+          * Image with Java amd64 binary: `projectnessie/nessie:${RELEASE_VERSION}-java-amd64\`
```

Comment:
JRE images do not necessarily use architecture as part of the tag. E.g. https://hub.docker.com/layers/library/eclipse-temurin/8u362-b09-jre-ubi9-minimal/images/sha256-672683d8b80920b4ea20c0ed812382d2d6d948dfb8767883fdcae5ed5113d857?context=explore has images for three architectures under the same tag...

Just wondering if `VERSION-java` vs. `VERSION-native` could be sufficient.

Reply:
See https://github.com/projectnessie/nessie/issues/6099#issuecomment-1431471217